documentation/docs/examples/cad-transform-override.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: cad-transform-override
+title: Override CAD nodes transforms
+description: Reveal supports overriding transforms of individual objects by providing tree indices to identify 3D nodes
+---
+
+import { DemoWrapper } from '@site/src/components/DemoWrapper';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Reveal supports overriding transforms of individual objects by providing [tree indices to identify 3D nodes.](../concepts.md)
+
+Note that any transformation override is applied with respect to the THREE.js world coordinate space.
+
+## Examples
+
+<DemoWrapper name="Cognite3DViewerDemo" />
+
+### Setting node transform
+
+Setting the transform of a given node is done by `Cognite3DModel.setNodeTransformByTreeIndex` and takes a tree index,
+position, rotation and a flag if this transformation also should be applied to children of the given node.
+
+
+
+```jsx runnable
+const position = new THREE.Vector3(10, 10, 0);
+const rotation = new THREE.Euler(0, 0, 0);
+model.setNodeTransformByTreeIndex(532261, {position, rotation}, true);
+```
+
+### Remove node transform
+
+Removing the transformation is done with `Cognite3DModel.resetNodeTransformByTreeIndex` and takes the given tree index,
+and a flag that determines if the node transform reset also should be applied to any child nodes of the given tree index.
+
+```jsx runnable
+model.resetNodeTransformByTreeIndex(532261, true);
+```
+
+### Animations

```

Comment:
We should also add an example using matrix instead of rotation + offset, and a comment about avoiding non-uniform scale.